src/pkg/packaging-tools/sharedFramework/sharedFramework.csproj

```diff
@@ -31,8 +31,8 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <!-- Don't error due to lack of runtime-specific deps, we're referencing those directly -->
     <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
-    <!-- Use a project-local packages folder -->
-    <RestorePackagesPath>$(IntermediateOutputPath)packages</RestorePackagesPath>
+    <!-- Use a project-local packages folder. 'p' rather than 'packages': stay under char limit. -->
```

Comment:
🤦‍♂  hopefully we don't actually rely on this too much: what about folks who clone under a deep path?

Reply:
I was tempted to use that exact emoji in the comment. Anyone with a deeper path than the build agent will probably fail, but I'm also concerned that some `**/*` somewhere will fail to pick some stuff up but end up succeeding with bad packages.

I can't find the build where I ran into the issue--I'll revert this and run again to see exactly what happens.

I tried to think of a shorter path but the issue is distinguishing it enough... right now it's
`C:\git\core-setup\bin\obj\win-x64.Release\Microsoft.WindowsDesktop.App\sfx\p` locally. Maybe
`C:\git\core-setup\bin\obj\sfx-packages\windowsdesktop` since we have the "short name" accessible and it doesn't seem likely to build two configurations in parallel on the same machine.

Reply:
Found the error:

```
error MSB3030: Could not copy the file "
F:\workspace\_work\1\s\bin\obj\win-arm64.Release\Microsoft.WindowsDesktop.App\sharedFx\packages\runtime.win-arm64.microsoft.windowsdesktop.app\3.0.0-preview6-27626-02\runtimes\win-arm64\lib\netcoreapp3.0\zh-Hans\System.Windows.Forms.Design.Editors.resources.dll
" because it was not found.
```

261 chars. At least it's not silent.

I intended to turn off the arm64/arm WD builds with this PR so the specific instance shouldn't apply, but I think I'll try the even shorter path above to help out devs with slightly longer paths anyway. (There are other errors with the arm/arm64 legs that I'm dealing with now that are probably related to disabling them for WD.)

Reply:
> At least it's not silent.

👏 thank you for checking.